internal/formats/syftjson/to_format_model.go

```diff
@@ -40,6 +46,85 @@ func ToFormatModel(s sbom.SBOM, applicationConfig interface{}) model.Document {
 	}
 }
 
+func toSecrets(data map[source.Coordinates][]file.SearchResult) []model.Secrets {
```

Comment:
`map[source.Coordinates][]file.SearchResult`

Is there any value in codifying these as concreate types beyond the assembly of their basic types?

Found as part of Sbom here:
https://github.com/anchore/syft/blob/4f0099583ab5b2030188de0b15367011b020957b/syft/sbom/sbom.go#L17-L25

`map[source.Coordinates][]file.SearchResult => someIndex`?

Only asking to see if additional methods could be added  which makes working with these types easier

Reply:
From our offline conversation: I'll try out making types for each of these "map" fields but probably won't add any helper functions for those types.

Reply:
I've added the new types didn't make any behavioral changes --the new types at the very least abstract what is in the type so if it changes later (say to a struct) then we don't need to update a bunch of function signatures.

Reply:
Hmm, well. When I updated the `file` package to use the new `sbom.File*` types, I got this:
```
```